client/.eslintrc

```diff
@@ -10,6 +10,8 @@
     "new-cap": [2, { "capIsNewExceptions": ["Set"] }],
     "react/require-default-props": "off",
     "react/no-array-index-key": "warn",
+    "import/no-extraneous-dependencies": ["warn", { devDependencies: true }],
+    "global-require": "off",
```

Comment:
Shall we only disable this in that file? cause we don't want global-require in other places.